cli/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "local-home-testing-cli",
+  "version": "0.0.1",
+  "description": "A command-line interface for interacting with Local Home SDK stubs.",
+  "private": true,
+  "author": "Google LLC.",
+  "license": "Apache-2.0",
+  "main": "./build/src/index.js",
+  "scripts": {
+    "compile": "tsc",
+    "test": "ava",
+    "clean": "gts clean",
+    "cli": "node ./build/src/index.js",
```

Comment:
ts-node?

Reply:
Since we're not using `ts-node` in other projects, I'm inclined to just use `node`.  If I'm not mistaken, the only difference would be the compilation step (and we `include` the build files).

Reply:
This does make the `cli` script dependent on `compile`, should we automatically run it before? or is it documented in the README? (one thing I'm afraid is that since the `cli` script is mainly used for development, one my forget to recompile between each modification).